docs/prepare-your-dev-environment.md

```diff
@@ -54,7 +54,9 @@ This document goes through the development dependencies one requires in order to
 
 ### Debian Packages
 
-Install the `libgpgme-dev` package.
+Install the `libgpgme-dev`, `libbtrfs-dev` and `libdevmapper-dev` packages.
```

Comment:
This matches my own testing when I was working on the [CodeQL stuff](https://github.com/Azure/ARO-RP/blob/2114a6c103600f814b710ba150e24ef3ec62ce50/.github/workflows/codeql-analysis.yml#L39), so 👍